portfolio/src/main/webapp/truths_and_lies.js

```diff
@@ -1,53 +1,192 @@
-    ['I am learning how to skateboard', '/images/two_truth_one_lie_imgs/skateboard.jpg'],
-    ['I was actually a TA for Google before I was an intern', '/images/two_truth_one_lie_imgs/Google.jpg'],
-    ['I nearly died when I ran my bike straight into a fence at full speed', '/images/two_truth_one_lie_imgs/bike.jpg'],
-    ['I worked at a mall aquarium', '/images/two_truth_one_lie_imgs/aquarium.jpg'],
-    ['I worked vaccuming buildings', '/images/two_truth_one_lie_imgs/vaccum.jpg'],
-    ['I won a gift card for knowing all the words to Ice Ice Baby', '/images/two_truth_one_lie_imgs/ice_ice_baby.jpg'],
-    ['I am born in the year of the dragon', '/images/two_truth_one_lie_imgs/dragon.jpg'],
-    ['I still like reading ridiculous \'group of teens save the world books\'', '/images/two_truth_one_lie_imgs/hunger_games.jpg'],
-    ['I don\'t like the taste of soda', '/images/two_truth_one_lie_imgs/soda.jpg'],
-    ['I have lived in LA but still haven\'t met someone famous', '/images/two_truth_one_lie_imgs/Los_Angeles.jpg'],
-    ['I trecked up 3,000 feet of ice unprepared because that\'s how it be', '/images/two_truth_one_lie_imgs/snow_mountain.jpg'],
-    ['I learned a language consisting of replacing all vowels with IG', '/images/two_truth_one_lie_imgs/language.jpg'],
-    ['I was ranked 7th in the state of Colorado for goals scored my junior year', '/images/two_truth_one_lie_imgs/soccer.jpg'],
-    ['I watched every episode of Avatar to write a paper my freshman year of college', '/images/two_truth_one_lie_imgs/last_airbender.jpg'],
-    ['I still struggle to know the difference between left and right', '/images/two_truth_one_lie_imgs/one_way.jpg'],
-    ['I actually like Country music', '/images/two_truth_one_lie_imgs/cowboy.jpg'],
-    ['I played football as a child', '/images/two_truth_one_lie_imgs/football.jpg'],
-    ['I wore a \'flying monkey\' costume to my fifth grade music show', '/images/two_truth_one_lie_imgs/wizard_of_oz.jpg'],
-    ['I am partially convinced we are living in a simulation', '/images/two_truth_one_lie_imgs/simulation.jpg'],
-    ['I am scuba diving certified', '/images/two_truth_one_lie_imgs/scuba_diving.jpg'],
-    ['I won a 5th grade D2 championship with a band of misfits', '/images/two_truth_one_lie_imgs/basketball.jpg'],
-    ['I recieved 14 Varsity Letters in High School', '/images/two_truth_one_lie_imgs/letter.jpg'],
-    ['I performed standup comedy in my fifth grade talent show', '/images/two_truth_one_lie_imgs/stand_up.jpg'],
-    ['I was terrified of a children\'s book called the Mosely Bug', '/images/two_truth_one_lie_imgs/child_book.jpg'],
-    ['My first crush was Sam from iCarly', '/images/two_truth_one_lie_imgs/nick.jpg'],
-    ['I would like to be a contestant on CBS Survivor', '/images/two_truth_one_lie_imgs/fire.jpg']
-]
+const truths =
+    [
+      [
+        'I wrestled at a juvenile detention school',
```

Comment:
Same comment about formatting as elsewhere, make sure to format before merging.

Reply:
Will do that from now on